js/misc/fileUtils.js

```diff
@@ -97,3 +97,17 @@ function loadInterfaceXML(iface) {
 
     return xml;
 }
+
+function getFileContents(filename) {
+    let contents = null;
+    try {
+        let [ok_, bytes] = GLib.file_get_contents(filename);
+        if (bytes instanceof Uint8Array)
+            contents = imports.byteArray.toString(bytes);
+        else
+            contents = bytes.toString();
```

Comment:
I don't understand why this instanceof check is necessary. Surely given we only support a single version of gjs at a time, the types returned by a particular function will always be the same.

In any case I think this entire function is unnecessary: use `Shell.get_file_contents_utf8_sync`.

![Screenshot from 2020-01-14 12-18-09](https://user-images.githubusercontent.com/86760/72343942-fb41b280-36c7-11ea-98ba-6a97e9d90010.png)

Reply:
Nice catch, I failed to find an existing function in js/ for that. As for the instanceof check, this was needed otherwise gjs would complain about using toString directly or something like that - this pattern is used in several other places in the shell code.

Anyway, updated to use `Shell.get_file_contents_utf8_sync`.